kvrocks.conf

```diff
@@ -953,5 +953,12 @@ rocksdb.rate_limiter_auto_tuned yes
 # Default: yes
 # rocksdb.avoid_unnecessary_blocking_io yes
 
+# Specifies the maximum size in bytes for a write batch in RocksDB.
+# If set to 0, there is no size limit for write batches.
+# This option can help control memory usage and manage large WriteBatch operations more effectively.
+# 
+# Default: 0 (no size limit)
```

Comment:
```suggestion
# Default: 0
```

It's the same thing as above. Personal opinion, it doesn't really matter.

Reply:
done, thank u